auto_sizing/targets.py

```diff
@@ -85,11 +90,31 @@ def _make_desktop_targets(self, target: Dict[str, str]) -> List[Segment]:
                 select_expr=clients_daily_sql,
             )
         )
-        Segment_list.append(client_type)
+
+        clients_last_seen = SegmentDataSource(
+            name="clients_last_seen",
+            from_expr="`moz-fx-data-shared-prod.telemetry.clients_last_seen`",
+        )
+        if target["user_type"] == "new":
```

Comment:
Before I review the actual code, one question: it looks like a lot of the definitions are written in code. So as we scale to additional apps, we'd have to write a new `_make_<app>_targets` function. This also makes it harder to identify where the logic is. Not saying this is a requirement to merging, this can be an improvement, but we should look into the ability to define all the Segment definitions outside the code, in a TOML file. Then the code just needs to read in and render the definitions. 

For example, in order to find out what existing user definition is for desktop, you'd have to know to look [here](https://github.com/mozilla/auto-sizing/blob/5bcd3465a63221346d116666cb893dcbce5c3974/auto_sizing/targets.py#L111), which is a bit buried.

Reply:
Just wanted to give this a +1. Agreed that it doesn't need to be a blocker to merging this PR, but we should definitely create a ticket to do this if there isn't one already. Should be able to link to @danielkberry's comment directly in the ticket.